usmqe/web/tendrl/mainpage/clusters/cluster_list/pages.py

```diff
@@ -4,22 +4,58 @@
 
 
 import copy
+import pytest
 
 from webstr.patternfly.contentviews import pages as contentviews
 import webstr.patternfly.dropdown.pages as dropdown
 
+from usmqe.gluster import gluster
+from usmqe.ceph import ceph_cluster
 import usmqe.web.tendrl.mainpage.clusters.cluster_list.models as m_cluster_list
 from usmqe.web.tendrl.mainpage.clusters.import_cluster_wizard.pages\
     import ImportCluster, ImportClusterSummary
 from usmqe.web.tendrl.auxiliary.pages import ListMenu
-from usmqe.web.tendrl.mainpage.tasks.pages import TaskDetails
+
+
+def check_hosts(hosts_list, page_hosts_list):
+    """
+    check if all hosts in host_list are present on the page (in page_host_list)
+
+    Parameters:
+        hosts_list (list): list of dictionaries
+                          {'hostname': <hostname>, 'release': <release>, ...
+        page_hosts_list (list): list representing lines in the page hosts list
+    """
+    for host in page_hosts_list:
+        found = False
+        for item_nr in range(len(hosts_list)):
+            if hosts_list[item_nr]['hostname'] in host.name:
```

Comment:
If I iterate directly on `hosts_list` then I cannot delete the item from the list easily.